src/YieldDisburser.sol

```diff
@@ -247,33 +239,26 @@ contract YieldDisburser is OwnableUpgradeable {
 
         BREAD.claimYield(BREAD.yieldAccrued(), address(this));
         uint256 projectsLength = projects.length;
-        (uint256[] memory projectDistributions, uint256 totalVotes) = _commitVotedDistribution();
-        if (totalVotes == 0) {
-            projectDistributions = new uint256[](projectsLength);
-            for (uint256 i; i < projectsLength; ++i) {
-                projectDistributions[i] = 1;
-            }
-            totalVotes = projectsLength;
-        }
 
         lastClaimedTimestamp = Time.timestamp();
```

Comment:
Currently this algorithm isn't sequenced with the cycles at all, so I assume we are going to be tolerant of some small gaps in between the cycles?